db/index.js

```diff
@@ -0,0 +1,11 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  user: 'root',
+  password: 'root',
```

Comment:
for real life practice I'd probably utilize your config.js file and hide your user and password information

Reply:
@chandradez Great suggestion, thanks!